runtime/src/accounts_db.rs

```diff
@@ -1339,11 +1339,18 @@ impl AccountsDB {
         hasher.result()
     }
 
-    pub fn include_owner_in_hash(slot: Slot) -> bool {
-        // Account hashing will be updated to include owner at this slot on the devnet.
-        // For testnet, it fully transitioned already thanks to eager rent collection,
-        // so, this check is irrelevant, strictly speaking.
-        slot >= 5_800_000
```

Comment:
Since devnet's ledger has been restarted, should this just return `true` ?  I'm seeing a bankhash mismatch when opening a development mode 1.2 ledger with 1.3, but it works fine with stable mode

Reply:
Confirmed returning `true` here works in development mode

Reply:
@t-nelson Thanks for looking into this! Due to #12068, the naming is a bit confusing. In short, `OperatingMode::Development` is named to `ClusterType::Devnet`. And `ClusterType::Development` is newly added on master/v1.3. So, `ClusterType::Developemnt` should completely not be abi-compatible with v1.2. Did you test _live_ (the so-called) devnet ledger after changing to return `true`?

> Since devnet's ledger has been restarted, should this just return true ?

I think exactly because of this (devnet was and still is under 5_800_000), this cannot just be `true` .

Hope we can get away with v1.2 in a week or so. :)

Reply:
> Since devnet's ledger has been restarted

Also, this restart _incorrectly_ happened under the `slot >= 5_800_000` condition. So, I think we must keep it around for a bit.

Reply:
Gotcha!  I think I'll just plumb some genesis flags through testnet automation so I can run the rolling upgrade as `stable`